lib/v2/screens/profile_screens/citizenship/citizenship_screen.dart

```diff
@@ -1,246 +1,21 @@
-                          rate: '${_visitors ~/ 100}/$required_visitors_invited',
-                          rateStyle: Theme.of(context).textTheme.subtitle1!,
-                        ),
-                        CircularProgressItem(
-                          icon: SvgPicture.asset('assets/images/citizenship/age.svg'),
-                          totalStep: required_account_age,
-                          currentStep: _age,
-                          circleRadius: 30,
-                          title: 'Account Age'.i18n,
-                          titleStyle: Theme.of(context).textTheme.subtitle3,
-                          rate: '$_age/$required_account_age',
-                          rateStyle: Theme.of(context).textTheme.subtitle1!,
-                        ),
-                        CircularProgressItem(
-                          icon: SvgPicture.asset('assets/images/citizenship/planted.svg'),
-                          totalStep: required_planted_seeds,
-                          currentStep: _seeds,
-                          circleRadius: 30,
-                          title: 'Planted Seeds'.i18n,
-                          titleStyle: Theme.of(context).textTheme.subtitle3,
-                          rate: '$_seeds/$required_planted_seeds',
-                          rateStyle: Theme.of(context).textTheme.subtitle1!,
-                        ),
-                        CircularProgressItem(
-                          icon: SvgPicture.asset('assets/images/citizenship/transaction.svg'),
-                          totalStep: required_seeds_transactions,
-                          currentStep: _transactions,
-                          circleRadius: 30,
-                          title: 'Transactions with Seeds'.i18n,
-                          titleStyle: Theme.of(context).textTheme.subtitle3,
-                          rate: '${_transactions}/$required_seeds_transactions',
-                          rateStyle: Theme.of(context).textTheme.subtitle1!,
-                        ),
-                        CircularProgressItem(
-                          icon: SvgPicture.asset('assets/images/citizenship/community.svg'),
-                          totalStep: required_resident_invited,
-                          currentStep: _users,
-                          circleRadius: 30,
-                          title: 'Invited Users'.i18n,
-                          titleStyle: Theme.of(context).textTheme.subtitle3,
-                          rate: '${_users ~/ 100}/$required_resident_invited',
-                          rateStyle: Theme.of(context).textTheme.subtitle1!,
-                        ),
-                      ],
-                    ),
-                  ],
-                );
-              default:
-                return const SizedBox.shrink();
-            }
-          },
-        ),
+        body: args.profile.status == 'visitor' ? const VisitorView() : const ResidentView(),
```

Comment:
I think this `status` from `ProfileModel` It would be better to use an `enum` instead of a `String`, because there is a lot of String distributed to equal the current status

Reply:
I agree, in follow up PR, the map this string to an Enum when we make the model